src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable-next-line no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://mcr-codes-weather-app.herokuapp.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  return axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
+    })
+    .catch((error) => {
+      const { status } = error.response;
+      if (status === 404) {
+        setErrorMessage("No such town or city, try again!");
+        console.error("Location is not valid", error);
```

Comment:
Since you are already displaying an error message to the user explaining why things are not working, these console logs could be removed
You could also remove the eslint no console exception at the top of the file as a result of this too